interface/backend/api/views.py

```diff
@@ -68,24 +68,27 @@ def walk(self, location, dir_name='root'):
     def get(self, request):
         """
         Return a sorted(by name) list of files and folders
-        in dataset in the form
-        {'directories': [
-            {
-                'name': directory_name1,
-                'children': [
-                    file_name1,
-                    file_name2,
-                    {
-                        'name': 'nested_dir_1',
-                        'children': [
-                            'file_name_1',
-                            'file_name_2',
-                            ....
-                        ]
-                    }
-                    ... ]
-            }, ... ]
-        }
+        in dataset
+
+        Format::
+        
```

Comment:
Removing the whitespaces here should make Travis pass :) (W293 blank line contains whitespace)

Reply:
Cheers, fixed!